docs/components/ComponentAnatomy.jsx

```diff
@@ -54,10 +76,16 @@ export const ComponentAnatomy = ({
     setTree(t);
   };
 
+  useEffect(() => {
+    waitForElement(".RenderVisualiser-component > *").then((element) => {
```

Comment:
`waitForElement` is querying `document`, it will get the wrong item? e.g. Looks like Button anatomy is showing the same information.

Reply:
Good spot! While I was fixing this I realised we don't even need the `waitForElement` function so I've removed it